docs/docs/routing.md

```diff
@@ -0,0 +1,238 @@
+---
+id: routing
+title: Routing
+permalink: docs/routing.html
+---
+
+Routing is the process of choosing what to render based on the browser's current URL.
+
```

Comment:
Also let's mention that in this guide, we'll build a simple routing solution from scratch, but there are popular community projects such as React Router.